### PR TITLE
Changed code to improve output for files under test/e2e/lifecycle

### PR DESCRIPTION
**Pull request by NikhilSharmaWe**
#### What type of PR is this?
Enhancement 

#### What this PR does / why we need it:
For better (more informative) output for developers when test fails. Changed test/e2e/lifecycle for this PR.
#### Which issue(s) this PR fixes:

Part of #105678

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by NikhilSharmaWe** [comment]
@pohly could you please review the changes made.

File: test/e2e/lifecycle/bootstrap/bootstrap_signer.go [hunk 1]
```diff
@@ -83,7 +84,9 @@ var _ = lifecycle.SIGDescribe("[Feature:BootstrapTokens]", func() {
 		cfgMap, err := f.ClientSet.CoreV1().ConfigMaps(metav1.NamespacePublic).Get(context.TODO(), bootstrapapi.ConfigMapClusterInfo, metav1.GetOptions{})
 		framework.ExpectNoError(err)
 		signedToken, ok := cfgMap.Data[bootstrapapi.JWSSignatureKeyPrefix+tokenID]
-		framework.ExpectEqual(ok, true)
+		if !ok{
```
Comment (pohly) [inline test/e2e/lifecycle/bootstrap/bootstrap_signer.go:87]:
```suggestion
		if !ok {
```

**Comment by NikhilSharmaWe** [comment]
@pohly made changes according to your review.

**Comment by pohly** [comment]
/retest

pull-kubernetes-integration failure looks like an unrelated test flake.

**Comment by pohly** [comment]
/assign @neolit123

For approval.

**Comment by neolit123** [comment]
thanks

/kind cleanup
/approve

**Pull request merged**